src/components/Person/Person.jsx

```diff
@@ -1 +1,18 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerType = (sex === 'f' && 'husband') || 'wife';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && (
+        <p className="Person__age">{`I am ${age}`}</p>
+      )}
+      <p className="Person__partner">
+        {(isMarried && `${partnerName} is my ${partnerType}`) || `I am not married`}
```

Comment:
the ternary operator can be used
```suggestion
        {isMarried
          ? `${partnerName} is my ${partnerType}`)
          : `I am not married`}
```